cmd/config_options.go

```diff
@@ -11,8 +11,9 @@ import (
 
 // ConfigOptions represent the persistent configuration flags of event-generator.
 type ConfigOptions struct {
-	ConfigFile string
-	LogLevel   string `validate:"logrus" name:"log level" default:"info"`
+	ConfigFile string `validate:"filepath" name:"config"`
+	LogLevel   string `validate:"logrus" name:"loglevel" default:"info"`
+	LogFormat  string `validate:"format" name:"logformat" default:"text"`
```

Comment:
A curiosity: why not using an OR?
(`eq=text|eq=json` if I don't remember it wrong)

Reply:
:+1: 
I just forgot about that. 
Let me check if it works

Reply:
Done by https://github.com/falcosecurity/event-generator/pull/42/commits/eec04ea752779246adfcb3000f25c926e6000485.
However, I kept "format" as an alias, so we can easily attach the translator.